src/_InputValidation.tsx

```diff
@@ -28,6 +28,7 @@ const InputValidation: React.FC<Props> = ({children, id, validationStatus, sx})
   return (
     <Text
       fontSize={0}
+      fontWeight="bold"
       sx={{
         alignItems: 'center',
```

Comment:
Should we move these inside the `sx` prop for consistency?
```suggestion
      sx={{
        fontSize: 0,
        fontWeight: "bold",
        alignItems: 'center',
```